joschahenningsen/TUM-Live
# 1006 upload camera lecture slides as vod

### Motivation and Context
Currently its not possible to add lecture slides and camera videos as dedicated video files.

### Description
Add input fields to allow the upload of camera and slides video too. Because the upload formular becomes a little
user unfriendly, it is refactored. Also, a jump to new added lectures and is introduced together with a little marker
that marks the new added lectures in the list.

### Steps for Testing
Prerequisites:
- 1 Admin
- 1 Course

1. Log in
2. Navigate to a Course Admin Page
3. Upload a new VoD with dedicated camera and slides video
4. Check if all videos are working
5. Upload a new VoD with combined stream only
6. Check if the VoD with combined stream is working

### Screenshots

https://github.com/joschahenningsen/TUM-Live/assets/17506411/7052c833-3b1d-4a25-acbe-c602929107d2
